src/renderer/components/atoms/text/text.stories.mdx

```diff
@@ -0,0 +1,354 @@
+<!-- Text.stories.mdx -->
+
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Text from './text';
+
+<Meta title="Atoms/Text" component={Text} />
+
+export const Template = (args) => <Text {...args} />;
+
+ ---
+
+# Text - interface
+
```

Comment:
I think it's more clear if you have a `# Text` title, and then a `## Usage for App UI` and another `## Usage for Docs generation` as subtitles

Reply:
done!